### PR TITLE
Ft question

### PR DESCRIPTION
**Pull request by NP-SON-250**
### What this PR does?
1. Adds question documentation
2. Adds changes on questions.controller and .routes
3. Adds changes on categories.routes

### CODE REVIEW

File: src/controllers/question.controller.js [hunk 1]
```diff
@@ -11,18 +11,18 @@ export const createAQuestion = async (req, res) => {
...
     if (!checkCategory) {
         return res.status(404).json({
             status: "404",
             message: "Category not found",
         });
     }
-    const checkQuestin = await Questions.findOne({ questionPrase: questionPrase });
-
+    const checkQuestin = await Questions.findOne({ questionPhrase: questionPhrase });
+    console.log(checkQuestin)
```
Comment (muhozajohn) [inline src/controllers/question.controller.js]: Remove Console

File: src/routes/question.routes.js [hunk 2]
```diff
@@ -10,10 +10,10 @@ import {
  } from "../controllers/question.controller";
 
  const questionRoute = express.Router();
- questionRoute.post("/:id",fileUpload.single("question"),createAQuestion);
+ questionRoute.post("/:id",Auth,fileUpload.single("question"),createAQuestion);
  questionRoute.get("/read/:id",viewAll);
```
Comment (muhozajohn) [inline src/routes/question.routes.js:14]: you can use default question routes

Comment (muhozajohn) [inline src/routes/question.routes.js:14]: same as here you use default question routes

**Review by muhozajohn — CHANGES_REQUESTED** [review]
Check Again and write clear code

File: src/controllers/question.controller.js (hunk 1 above)
Comment (NP-SON-250) [inline src/controllers/question.controller.js]: Removed bro

**Review by muhozajohn — APPROVED** [review]

**Pull request merged**